### ISSUE TITLE
Make tensorflow-macos version match to build wheel successfully on Apple Silicon.

### ISSUE THREAD

**Issue by tuzhucheng** [tensorflow/text#1234]
To make the wheel successfully build on Apple Silicon machines, we can do the following:

For the 2.15 release, the `tensorflow-macos` version should be 2.15.0: https://github.com/tensorflow/text/blob/2.15/oss_scripts/configure.sh.

For the 2.14 release, the `tensorflow-macos` version should be 2.14.0: https://github.com/tensorflow/text/blob/2.14/oss_scripts/configure.sh.

**Comment by tuzhucheng** [comment]
Or we can directly install `tensorflow` even for macOS since starting from 2.13 it appears the arm64 wheels are included with `tensorflow` on pypi: https://pypi.org/project/tensorflow/2.15.0/#files.

**Comment by yarri-oss** [comment]
Yes, and we'd like to first put better testing in place and then update the references in `configure.sh` to `tensorflow` because it's supports Aarch64 wheels with an udpated `setup.py` shim.

**Comment by markblee** [comment]
Hi, is there any update or ETA for this? Thanks.
